src/main/java/org/cryptomator/ui/traymenu/AwtTrayMenuController.java

```diff
@@ -1,5 +1,6 @@
 package org.cryptomator.ui.traymenu;
 
+import dagger.internal.Preconditions;
```

Comment:
These are the wrong `Preconditions`. Should be sth like `com.google.common.`

Reply:
updated